CodeEdit/WorkspaceView.swift

```diff
@@ -28,7 +28,8 @@ struct WorkspaceView: View {
     var body: some View {
         NavigationView {
```

Comment:
Now that you've removed `NavigationLink`, `NavigationView` here is not necessary. Can replace it with `HSplitView`?

Reply:
NavigationView has the benefit of giving us the translucent background in side bar for free.

Reply:
Oh, get that.